docs/migration/migrate_8_0.asciidoc

```diff
@@ -50,41 +50,41 @@ for example, `logstash-*`.
 
 [float]
 ==== Default logging timezone is now the system's timezone
-*Details:* In prior releases the timezone used in logs defaulted to UTC.  We now use the host machine's timezone by default.
+*Details:* In prior releases the timezone used in logs defaulted to UTC. We now use the host machine's timezone by default.
 
-*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a date modifier:
+*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a <<date-format, date modifier>>:
 [source,yaml]
 -------------------
 logging:
   appenders:
     console:
-      kind: console
+      type: console
       layout:
-        kind: pattern
+        type: pattern
         pattern: "%date{ISO8601_TZ}{UTC}"
 -------------------
 See https://github.com/elastic/kibana/pull/90368 for more details.
 
 [float]
 ==== Responses are never logged by default
-*Details:* Previously responses would be logged if either `logging.json` was true, `logging.dest` was specified, or a `TTY` was detected.
+*Details:* Previously responses would be logged if either `logging.json` was true, `logging.dest` was specified, or a `TTY` was detected. With the new logging configuration, these are provided by a dedicated <<dedicated-loggers,logger>>.
 
-*Impact:* To restore the previous behavior, in kibana.yml enable `debug` logs for the `http.server.response` context under `logging.loggers`:
+*Impact:* To restore the previous behavior, in `kibana.yml` enable `debug` for the <<request-response-logger,`http.server.response`>> logger:
```

Comment:
Ditto re content re-used outside the Kibana book:
```suggestion
*Impact:* To restore the previous behavior, in `kibana.yml` enable `debug` for the {kibana-ref}/logging-service.html#request-response-logger[`http.server.response`] logger:
```

Reply:
See https://github.com/elastic/kibana/pull/94993#discussion_r597338769